src/data/utility/Util.js

```diff
@@ -225,3 +227,30 @@ export const mergeArraysWithObjects = (a = [], b = [], prop, mergeProp) => {
 
   return updatedArray;
 };
+
+/**
+ * It converts timestamps to UTC based on timezone from user identity.
```

Comment:
Isn't it, it converts UTC to the users' timezone?

Reply:
@maartenhemmes I get timezone setting from user's identity, and use it in transformation, so basically it does what it should, right?

Reply:
It doesn't convert it to UTC, it should convert it to user' timezone FROM UTC

Reply:
yes, this is what it does, I think I had wrong descriptions there